CHANGELOG.md

```diff
@@ -1,14 +1,49 @@
 # Changelog
 All notable changes to this project will be documented in this file. This CHANGELOG roughly follows the guidelines from [www.keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [v0.11.0]
+### Added
+- Add test for double square brackets to katex-spec (#1956)
+- Add option to render only MathML so that its visible (#1966)
+- Support {smallmatrix}, {subarray}, and \substack (#1969)
+- Enable minRuleThickness in rendering options (#1964)
+- Add \plim (#1952)
+- Support Unicode \digamma (#2010)
+- [breaking] trust setting to indicate whether input text is trusted (#1794)
```

Comment:
Maybe we should move this to the top (because breaking), and add mention (e.g. subbullet) that use of `\url` and `\href` will now break without tweaking this setting?

Reply:
Good idea.